modules/dnn/test/test_layers_1d.cpp

```diff
@@ -245,4 +245,34 @@ INSTANTIATE_TEST_CASE_P(/*nothing*/, Layer_Elemwise_1d_Test, Combine(
 /*operation*/           Values("div", "prod", "max", "min", "sum")
 ));
 
+typedef testing::TestWithParam<tuple<int, std::vector<int>>> Layer_Const_Test;
+TEST_P(Layer_Const_Test, Accuracy)
+{
+    int dims = get<0>(GetParam());
+    std::vector<int> input_shape = get<1>(GetParam());
+
+    LayerParams lp;
+    lp.type = "Const";
+    lp.name = "ConstLayer";
+
+    Mat constBlob = Mat(dims, input_shape.data(), CV_32F);
+    cv::randn(constBlob, 0.0, 1.0);
+    Mat output_ref = constBlob.clone();
+
+    lp.blobs.push_back(constBlob);
+    Ptr<Layer> layer = ConstLayer::create(lp);
+
+    std::vector<Mat> inputs; // No inputs are needed for a ConstLayer
+    std::vector<Mat> outputs;
+    runLayer(layer, inputs, outputs);
```

Comment:
insert `ASSERT_EQ(outputs.size(), 1)`

Reply:
Signature for `_EQ` is `ASSERT_EQ(expected_reference, actual_result);`.
Correct order of parameters are required to emit valid error messages.

Reference: https://github.com/opencv/opencv/blob/4.0.0/modules/ts/include/opencv2/ts/ts_gtest.h#L8196-L8200

```
GTEST_API_ AssertionResult EqFailure(const char* expected_expression,
                                     const char* actual_expression,
                                     const std::string& expected_value,
                                     const std::string& actual_value,
                                     bool ignoring_case);
```